### PR TITLE
[DOCS] Added endpoints documentation to ReadMe

### PR DESCRIPTION
**Pull request by josimar-silva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,219 @@
...
+## Webhooks
+
+### Adyen
+
+Webhook used by Adyen.
+
+<span style="color:#33cc33">POST</span> <span style="color:#1a75ff">/webhook/adyen</span>
+
+
+### Braspag
```
Comment (ferrl) [inline README.md]: I guess we can ✂️ this for now.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,219 @@
...
+
+## Transaction History
+|Property|Description|
+|-----|-----|
+|ID|Transaction's history unique identifier.|
+|From|Transaction's **previous** state.|
+|To|Transaction's **current** state.|
+|Request|Payload sent to a payment gateway.|
+|Response|Payload received from a payment gateway.|
+|Type|The type of caller of initiate the state change. Can be **HTTP**, **Webhook** and **manual**.|
```
Comment (ferrl) [inline README.md]: We discussed a `Success` bool. Are we going to add it?

File: README.md (hunk 1 above)
Comment (josimar-silva) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (josimar-silva) [reply README.md]: Yes. I forgot to add it :man_facepalming:.

**Pull request merged**